### ISSUE TITLE
is it possible to define outputs without labels and losses?

### ISSUE THREAD

**Issue by ophiry** [keras-team/keras#8032]
In a typical scenario with multiple outputs, each output has a related label, and a loss function.

However, in some cases it's desirable to return additional data that isn't used for training (for example: attention mask)

currently I'm adding dummy labels and dummy loss functions, but I was wondering if there's a cleaner way

**Comment by harshini-gadige** [comment]
Hi

This issue isn't related to a bug/enhancement/feature-request or other accepted types of issue.

To ask questions, please see the following resources :

https://keras.io/
https://keras.io/#support
https://gitter.im/Keras-io/Lobby

Thanks!

If you think I made a mistake, please open another issue.
